src/agent/api.rs

```diff
@@ -75,6 +75,18 @@ impl AgentApi {
         }
     }
 
+    fn capabilities(&self) -> Capabilities {
```

Comment:
It would be nice to have a way to override the list of capabilities when starting the agent from the CLI. That would allow easier local testing.

Reply:
Is an environment variable ok? Maybe `CRATER_AGENT_CAPABILITIES` with a comma-separated list?

Reply:
Any reason why you would prefer an environment variable? All of the current Crater configuration is done through CLI args, so being consistent would be nice.

Reply:
I wasn't sure which CLI command should set this parameter. Is `crater agent` responsible for telling the agent to start requesting experiments from the server?

Reply:
Yes, everything in `src/agent` is only used by the `crater agent` command.

Reply:
Implemented [here](https://github.com/rust-lang/crater/pull/450/files#diff-dba02791b4baf8a328939029ac920005R270). I would have used an `Option<Vec<String>>` so that passing `--capabilities=` would skip the default capabilities, but this is not supported until `structopt-0.3`.